### PR TITLE
Add bilat_trade to iomath

### PR DESCRIPTION
**Pull request by rich-wood**
Adding functionality to iomath to pull out bilateral trade data. I have not updated the mriosystem or calc_all to include this routine in there yet. Perhaps check the naming of output variables for consistency (noting that "import" and "export" are used in a "net" sense elsewhere in iomath)

### CODE REVIEW

File: pymrio/tools/iomath.py [hunk 1]
```diff
@@ -410,3 +410,44 @@ def calc_accounts(S, L, Y, nr_sectors):
...
+        Symmetric input output table (flows)
+    Y : pandas.DataFrame
+        final demand with categories (1.order) for each country (2.order)
+
+    Returns
+    -------
+    pandas.DataFrame
+        bilateral trade flows as a matrix showing region and sector of origin in rows, and region of import in columns
+                
+   NOTE: ONLY IMPLEMENTED FOR DataFrame right now
```
Comment (konstantinstadler) [inline pymrio/tools/iomath.py:430]: In numpy docstrings (which you are using here, thats great), the notes would come before the Parameters like:

Notes
---------

ONLY IMPLEMENTED FOR DataFrame right now

Parameters
---------------
...

small detail, if you want to change.

**Review by konstantinstadler — CHANGES_REQUESTED** [review]
Many thanks for that! I will merge that in when I fixed the other error. 
I just requested a small change, if you are up to it (basically you change it in you repo and just push again, it will automatically update the merge request)

File: pymrio/tools/iomath.py (hunk 1 above)
Comment (rich-wood) [reply pymrio/tools/iomath.py]: done - hope that is OK!

**Comment by konstantinstadler** [comment]
thanks a lot, I will integrate it into the pymrio object

**Pull request merged**